foxmlToSolr.xslt

```diff
@@ -38,6 +38,12 @@
   <xsl:param name="TRUSTSTOREPATH" select="repositoryName"/>
   <xsl:param name="TRUSTSTOREPASS" select="repositoryName"/>
 
+  <!--
+    Parameter(s) from custom_parameters.properties.
+  -->
+  <xsl:param name="index_ancestors" select="false()"/>
+  <xsl:param name="index_compound_sequence" select="false()"/>
```

Comment:
Not terribly concerned, but... potential trap here, with the coded default being different from [that being made in core](https://github.com/discoverygarden/basic-solr-config/pull/196/files#diff-3d413d43a0ca821813d2777605442fd3R47)... paranoia?